src/main/java/org/spongepowered/common/registry/type/NotePitchRegistryModule.java

```diff
@@ -57,10 +56,6 @@
 
     @Override
     public void registerDefaults() {
-        RegistryHelper.mapFields(NotePitches.class, input -> {
-            NotePitch pitch = new SpongeNotePitch((byte) this.notePitchMap.size(), input);
-            this.notePitchMap.put(input.toLowerCase(Locale.ENGLISH), pitch);
-            return pitch;
-        });
+        NoteUtils.pitchMappings.values().forEach(notePitch -> this.notePitchMap.put(notePitch.getId().toLowerCase(Locale.ENGLISH), notePitch));
```

Comment:
Yes

-------- Ursprüngliche Nachricht --------
Von: Seppe Volkaerts <notifications@github.com> 
Datum: 04.05.2016  10:55  (GMT+01:00) 
An: SpongePowered/SpongeCommon <SpongeCommon@noreply.github.com> 
Cc: ST-DDT <ST-DDT@gmx.de>, Mention <mention@noreply.github.com> 
Betreff: Re: [SpongePowered/SpongeCommon] Use the same NotePitch instances in
  NoteUtils and the registry module. (#699) 

In src/main/java/org/spongepowered/common/registry/type/NotePitchRegistryModule.java:
> @@ -57,10 +56,6 @@
>  
>      @Override
>      public void registerDefaults() {
> -        RegistryHelper.mapFields(NotePitches.class, input -> {
> -            NotePitch pitch = new SpongeNotePitch((byte) this.notePitchMap.size(), input);
> -            this.notePitchMap.put(input.toLowerCase(Locale.ENGLISH), pitch);
> -            return pitch;
> -        });
> +        NoteUtils.pitchMappings.values().forEach(notePitch -> this.notePitchMap.put(notePitch.getId().toLowerCase(Locale.ENGLISH), notePitch));

@ST-DDT Do you mean like:

NoteUtils.pitchMappings.values().forEach(notePitch -> 
        this.notePitchMap.put(notePitch.getId().toLowerCase(Locale.ENGLISH), notePitch));

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub